scripts/v8/clang-wrap.sh

```diff
@@ -24,6 +24,12 @@ while (( "$#" )); do
       -Wmax-tokens) ;;
       # Prevents -stdlib=libc++ from working
       -no-canonical-prefixes) ;;
+      # Avoid debugging issues by making source paths absolute
```

Comment:
That will avoid debug issues for a local build, but will create them for someone using the .deb with a source checkout.

Reply:
The debug info always contains absolute paths, this just changes the way they are stored. Before, it was a combination of comp_dir (compilation dir) + name (relative source path) and now name is absolute on its own. Both are fine and equally bad in theory, but the latter has the advantage that it is more compatible with some debuggers like lldb (or vscode-lldb?), at least from my experience. In any case, if you really want to debug V8 then a local build is still the best option as you get the matching source code as well, in the right location. The main point of having a debug variant in Azure Storage is to allow running V8 with all debug checks enabled, allow it to run together with a SAN-enabled build of CCF, and get stack traces and resolved symbols. Actual source-level debugging is not a goal.
If CCF intends to release debuggable .debs at some point then IMO this requires more thoughtful planning and testing to make sure it actually works as expected.

Reply:
Ok, I see.

> If CCF intends to release debuggable .debs at some point then IMO this requires more thoughtful planning and testing to make sure it actually works as expected.

Agreed, this entirely aspirational at this point.